src/api/spec/jobs/bs_request_action_webui_infos_job_spec.rb

```diff
@@ -45,11 +45,14 @@
 
     context 'with non existing target project' do
       let(:request) do
-        create(:bs_request_with_submit_action,
-               source_project: source_project.name,
-               source_package: source_package.name,
-               target_project: 'does-not-exist',
-               target_package: target_package.name)
+        request = build(:bs_request_with_submit_action,
+                        source_project: source_project.name,
+                        source_package: source_package.name,
+                        target_project: 'does-not-exist',
+                        target_package: target_package.name)
+        # avoid save!
+        request.save
```

Comment:
this example is a little bizarre - and basically showing off why the PR got so large. Previously the actions were added behind the request's back. And as the really tricky validations are not in the action but in the request, this lead to all kind of 'impossible' requests created in rspec.